CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributing
+
+## When to submit an ADR
+
+The purpose of an ADR is to record an over-arching technical decision made by the development team in the Archivematica 
+project. An ADR is akin to a policy in other contexts- it describes a way that future features or changes should be 
+implemented, rather than describing the features themselves (to submit a feature idea, head to the Issues repo).  
+Developers will use ADRs to understand how code should be written and analysts and end-users to understand implementation 
+decisions.
+
+Examples of situations when an ADR is appropriate:
+
+* A change to how code is formatted 
+* Major architectural changes (e.g. bringing together or separation of components, upgrade of major dependencies, etc.)
+* Deprecation of existing features
+
+ADRs may be written retroactively to make a record of core features or functionality.
+
+## How to submit an ADR
+
+ADRs must be submitted as a pull request to this repo. To get started, first review currently accepted ADRs, then use the 
+template to write yours.
+
+ADRs can be submitted in WIP form if desired but in all cases should be submitted with the status “proposed.”
+
+While ADRs are often technically focused, it’s good practice to write them in plain, understandable language (The 
+Archivematica docs wiki provides good guidance on this subject).
+
+When you submit an ADR be sure to assign at least one reviewer. If you’re concerned about understandability you might
+include a reviewer of a different background than your own (e.g. assign an analyst as a reviewer if you are a developer 
+and vice versa).
+
+## How an ADR is reviewed and merged
+
+The purpose for code review of an ADR is:
+
+* clarity of language
+* formatting
+* ADR number collisions 
+
+Once the ADR is approved it can be merged by the submitter. While the code review ensures understandability of the ADR, 
+all Archivematica team and community members are encouraged to participate in discussion about the proposed changes. 
+
+When merging remember also to update the Accepted log.
```

Comment:
Will this need to be a PR as well?

Reply:
My thinking is that when you submit a new ADR as a PR, it should also include a change to the log. Open to other suggestions though!

Reply:
If you should include a change to the log as part of the ADR PR, this should be noted in the "How to submit an ADR" section above.

Reply:
Done.